docs/developers/contracts/technical-references/tick-ratio.md

```diff
@@ -1,100 +1,96 @@
 
-To put it simply, ticks are a way of sorting offers by price (into [tick bins](#3-tick-bins)). All the offers in a tick bin have the same tick. We also need a way to know where are the previous and next best bins: that's what [`tickSpacing`](#4-tickspacing) is for.
+$$
+price_{side \in \{asks, bids\}}(tick) = \begin{cases}
+   ratio = wants/gives = 1.0001^{tick} &\text{if } side \equiv asks \\
+   1/ratio = gives/wants = 1.0001^{-tick} &\text{if } side \equiv bids.
+\end{cases}
+$$
 
-:::danger Important
-Ticks are derived from the [ratio](#ratio).
-:::
-
-[?? To add a solidity example about `tickFromRatio()` ??]
-
-:::info Note
-We will briefly mention the notions of "tick tree" and "tick bins", however note that as a developer, they won't have much of an impact with your experience using Mangrove. What you really want to focus on is **ticks**.
-:::
-
-Let's break it down a bit.
+Note that the relationship between ratio and price **depends on which side** of the book you are looking at:
 
-### 1. Order book basics
+* On the _asks_ side, it's fairly natural: The price is the ratio $wants/gives$ between the amount $gives$ of base tokens promised by an offer and the amount $wants$ of quote tokens it requests.
 
-Mangrove's order book contains real-time, continuously updated lists of offers for trading pairs ([markets](./taking-and-making-offers/offer-list.md)). The book is organized by price levels (ticks), with the highest bid (buy order) and the lowest ask (sell order) sitting at the top of their respective side of the book ([offer lists](./taking-and-making-offers/offer-list.md)).
+* On the _bids_ side, however, the price is the reciprocal of the $wants/gives$ ratio, i.e. price = $gives/wants$. This is because a bid wants base tokens and offers quote tokens and thus the ratio $wants/gives$ is in `base/quote`.
 
-### 2. Tick tree
 
-As previously said, offers are stored in bins, which are sorted in a "tick tree". Thanks to this tree structure, **offer operations (insert, update, and retract) take constant gas** (the height of the tree is fixed).
+:::info Example
+On a WETH/DAI market:
+* The price would be expressed in DAI per WETH (DAI/WETH)
+    * Eg. 2,224 DAI/WETH
+* On the asks side WETH-DAI (selling WETH or buying DAI), the ratio equals the price and has unit DAI/WETH
+    * Eg. 2,224 DAI/WETH
+* On the bids side DAI-WETH (buying WETH or selling DAI), the ratio equals 1/price and has unit WETH/DAI.
+    * Eg. 0.0004496 WETH/DAI
+:::
 
-### 3. Tick bins
 
-Below the bottom of the tree are tick bins - they are laid in sequence. In the context of an [offer list](./taking-and-making-offers/offer-list.md), each tick bin has an associated tick (and a tick determines a price).
+:::warning Beware decimals!
+Note that ratio and price are in raw numbers, ie, disregarding token decimals. Care must be taken when converting raw prices and ratios to their human readable counterparts with decimals.
```

Comment:
We usually talk about prices in human readable, Big decimals, and often about ratios as raw. At least in the SDK (in solidity everything is raw). So I think this comment is a bit misleading.

I think the easiest is to let the docs be in human readable numbers and mention that solidity works with raw values, and ratios are mostly used there.

Reply:
In https://github.com/mangrovedao/mangrove-docs/pull/203 I've moved and revised this warning + added a section that expands on decimals.

I had also hoped (as discussed out-of-band) to discuss conventions in the SDK and docs more generally. But I didn't find a good way to do this, so I only covered what was relevant for this page.